paper.md

```diff
@@ -0,0 +1,74 @@
+---
+title: 'CLOVER: Community-scale sustainable energy system modelling'
+tags:
+ - Python
+ - energy access
+ - minigrid
+ - renewable energy
+ - sustainable development
+authors:
+ - name: Philip Sandwell
+  orcid: 0000-0003-1117-5095
+  affiliation: "1, 2"
+  corresponding: true
+- name: Benedict Winchester
+  orcid: 0000-0002-2880-1243
+  affiliation: "2, 3"
+- name: Hamish Beath
+  orcid: 0000-0002-5124-9143
+  affiliation: "1, 2"
+- name: Jenny Nelson
+  orcid: 0000-0003-1048-1330
+  affiliation: "1, 2"
+affiliations:
+ - name: Department of Physics, Blackett Laboratory, Imperial College London, SW7 2AZ, United Kingdom
+   index: 1
+ - name: Grantham Institute - Climate Change and the Environment, Imperial College London, SW7 2AZ, United Kingdom
+   index: 2
+ - name: Department of Chemical Engineering, Imperial College London, SW7 2AZ, United Kingdom
+   index: 3
+date: 19 August 2022
+bibliography: paper.bib
+---
+
+# Summary
+
+Sustainable Development Goal 7 aims to provide sustainable, affordable, reliable and modern energy access to all by 2030. In order for this goal to be achieved, sustainable energy interventions in developing countries must be supported with design tools which can evaluate the technical performance of energy systems as well as their economic and environmental impacts.
+
+CLOVER (Continuous Lifetime Optimisation of Variable Electricity Resources) is a software tool for simulating and optimising community-scale energy systems, typically minigrids, to support energy access in developing countries. CLOVER can be used to model electricity demand and supply at an hourly resolution, for example allowing users to investigate how an electricity system might perform at a given location. CLOVER can also identify an optimally-sized energy system to meet the needs of the community under specified constraints, for example a system which provides a desired level of reliability at the lowest cost of electricity. CLOVER can provide an insight into the technical performance, costs, and environmental impact of a system, and allow the user to evaluate many different scenarios to decide on the best way to provide sustainable, affordable and reliable electricity to a community.
```

Comment:
Replace "under specified constraints, for example a system"
With "under specified constraints. For example, a system"
Reason: Comma needed after "for example" and a sentence break is always nice

Reply:
Included as suggested